js/classes/entities/midground/weapon/sword.js

```diff
@@ -112,6 +112,12 @@ class Sword {
                     if (bb.collide(entity.boundingBox)) {
                         entity.takeDamage(this.damage * CHAD.damageMultiplier);
                         this.hasHit = true;
+
```

Comment:
Why is this commented out? Might be good to keep it in if it works until you find a better solution.